_episodes/01-why-bother-performance.md

```diff
@@ -164,11 +159,12 @@ If these terms are unfamiliar now, don't worry as we will cover them over the du
 > 4. Use GPUs instead of CPUs
 > 5. Splitting code up into smaller segments
 >
-> > 1. Yes, MPI can enable you to split your code into multiple processes distributed over multiple cores, known as parallel programming.
-> > 2. Yes, like MPI this is also parallel programming, but deals with threads, by splitting a process into multiple threads, each thread using a single core.
-> > 3. Yes, that is their purpose. Different libraries run on different architectures however.
-> > 4. Yes, GPUs are better at handling multiple tasks, whereas a CPU is better running singular tasks quickly.
-> > 5. No, if you have a simulation that needs to be run from start to completion, splitting the code into segments won't be of any benefit and will waste compute resources.
+>> ## Solution
+>> 1. Yes, MPI can enable you to split your code into multiple processes distributed over multiple cores, known as parallel programming.
+>> 2. Yes, like MPI this is also parallel programming, but deals with threads, by splitting a process into multiple threads, each thread using a single core.
+>> 3. Yes, that is their purpose. Different libraries run on different architectures however.
+>> 4. Yes, GPUs are better at handling multiple tasks, whereas a CPU is better running singular tasks quickly.
+>> 5. No, if you have a simulation that needs to be run from start to completion, splitting the code into segments won't be of any benefit and will waste compute resources.
```

Comment:
Well, I disagree a little bit here depending on what you mean by splitting into segments. Compilers are quite clever, making code more read-able through refactoring will most likely not waste resources because of this. What it will do is save *human resources* for someone who comes along and wants to modify the code (including the original developer). Human resources are far more valuable than CPU resources, so even if this came with a 10% performance price tag it is worth it,